### PR TITLE
More sorting in various places

### PR DESCRIPTION
**Pull request by jisookim0513**

### CODE REVIEW

**Comment by drcrallen** [comment]
General question here: Are all the places you do sorting on long or integer types?

If they are on string natural lexographic sorting might not be what you want.

File: app/kafka/manager/utils/AdminUtils.scala [hunk 1]
```diff
@@ -247,8 +247,11 @@ class AdminUtils(version: KafkaVersion) {
     checkCondition(topicsWithoutReadVersion.isEmpty, TopicErrors.NoReadVersionFound(topicsWithoutReadVersion.mkString(", ")))
 
     // topicAndReplicaList is sorted by number of partitions each topic has in order not to start adding partitions if any of requests doesn't work with newNumPartitions
-    for ((topic, replicaList) <- topicAndReplicaList) {
-      addPartitions(curator, topic, newNumPartitions, replicaList, brokerList, readVersions.getOrElse(topic,-1))
+    for {
+      (topic, replicaList) <- topicAndReplicaList
+      readVersion = readVersions(topic)
+    } {
+      addPartitions(curator, topic, newNumPartitions, replicaList, brokerList, readVersion)
```
Comment (drcrallen) [inline app/kafka/manager/utils/AdminUtils.scala:254]: This seems out of scope for this PR, is it supposed to be part of sorting?

**Comment by drcrallen** [comment]
Also, why is sorting needed?

File: app/kafka/manager/utils/AdminUtils.scala (hunk 1 above)
Comment (jisookim0513) [inline app/kafka/manager/utils/AdminUtils.scala:254]: Oh it's not, it's actually a modification that belongs to the previous commit that enables adding partitions to multiple topics at once.

**Comment by jisookim0513** [comment]
@drcrallen I sort partition numbers and topic names, which are integer and string. Also, sorting is nice to see the information on page because Kakfa Manager console displays the sequence directly. This PR addresses Max's inquiries for UI improvement.
